Firestore/Source/Public/FIRQuery.h

```diff
@@ -268,10 +242,69 @@ addSnapshotListenerWithIncludeMetadataChanges:(BOOL)includeMetadataChanges
  *
  * @return The created `FIRQuery`.
  */
-// clang-format off
 - (FIRQuery *)queryWhereFieldPath:(FIRFieldPath *)path
                     arrayContains:(id)value NS_SWIFT_NAME(whereField(_:arrayContains:));
-// clang-format on
+
+/**
+ * Creates and returns a new `FIRQuery` with the additional filter that documents must contain
+ * the specified field, the value must be an array, and that array must contain at least one value
+ * from the provided array.
+ *
+ * A query can have only one arrayContains filter and it cannot be combined with arrayContains or
```

Comment:
"in" within this statement is unquoted and when bare it looks weird. We should probably refer to this as an "in filter". How about putting `arrayContains` and `in` within backticks?

Oh actually, this is an `arrayContainsAny` filter anyway, not `arrayContains`. Please update.

Also this statement is slightly redundant--it says you can only have one arrayContains and that you can't combine it with arrayContains. If this applies to `arrayContainsAny` (which I think is true), how about:

> A query can have only one `arrayContainsAny` or `in` filter.